Makefile

```diff
@@ -121,3 +126,19 @@ help: ## Show this help.
 		if (/^[a-zA-Z_-]+:.*?##.*$$/) {printf "    ${YELLOW}%-30s${GREEN}%s${RESET}\n", $$1, $$2} \
 		else if (/^## .*$$/) {printf "  ${CYAN}%s${RESET}\n", substr($$1,4)} \
 		}' $(MAKEFILE_LIST)
+
+build_push: build
+    ifeq ($(DOCKER_REGISTRY),)
+        $(error DOCKER_REGISTRY is required. Please provide a value.)
+    endif
+	@docker buildx build -f build/Dockerfile --progress plain --output type=image,name=$(DOCKER_REGISTRY)/$(BINARY_NAME):latest,push=true .
+
+kube_deploy:
+	@if [ -z $(KUBECONFIG) ]; then \
+        KUBECONFIG=$$HOME/.kube/config; \
+    fi;
+	@KUBECONFIG=$(KUBECONFIG) helm upgrade --install $(APPNAME) -n$(NAMESPACE) ./deploy/sigma --set image.repository=$(REPOSITORY),image.tag=$(TAG),mysql.auth.rootPassword=$(RANDOM_PASSWORD),mysql.auth.password=$(RANDOM_PASSWORD),redis.auth.password=$(RANDOM_PASSWORD),minio.secretKey=$(RANDOM_PASSWORD)
+
+kube_undeploy:
+	@KUBECONFIG=$(KUBECONFIG)
+	@helm uninstall $(APPNAME) -n$(NAMESPACE)
```

Comment:
`helm --kubeconfig path/to/kubeconfig` is better here, same as the `kube_deploy`.